### PR TITLE
changelog: update before release

### PR DESCRIPTION
**Pull request by kyessenov**
Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

**Review by alecholmez — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,25 @@
 # Changelog
 
-## Release v0.9.8
+## Release v0.9.9
 
 ### Added
+
 - Support for setting TTL on resources and configuring resource heartbeating
```
Comment (snowp) [inline CHANGELOG.md:7]: Maybe remove this line if this is covered by L9?

Comment (kyessenov) [reply CHANGELOG.md:7]: Done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,25 @@
...
+- Incremental simple cache implementation (#411)
+
+### Changed
+
+- Envoy APIs are at b6039234e526eeccdf332a7eb041729aaa1bc286
+- Update dependencies to use `cncf/xds` instead of `cncf/udpa` (#404)
+- Log ignoring a watch at warn level (#352)
+- Go 1.16 compatibility fixes (#409)
+- Removed support for V2 Envoy APIs in the server (#415)
+- Fix a potential goroutine leak in stream handler (#430)
```
[the remark this answers is not in the document]
Comment (kyessenov) [reply CHANGELOG.md:20]: Done.

**Review by snowp — APPROVED** [review]
Thanks!

**Review by alecholmez — APPROVED** [review]
Looks good!

**Pull request merged**